src/js/edu-benefits/config/migrations.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash/fp';
+
+export default [
+  // 0 -> 1, we have split the edu bundle into form-specific bundles with a new urlPrefix
+  // and this replaces the saved returnUrl if necessary
+  (savedData) => {
+    const savedReturnUrl = _.get('return_url', savedData.metadata);
+
+    if (savedReturnUrl.includes(savedData.formId)) {
```

Comment:
The formId doesn't necessarily match the url prefix. For the 1990, the url prefix is 1990 but the form id is 22-1990.

I think you can make this a standalone function that takes a formId and returns the migration function. You can use that for each form's migration.

So like:

```js
export function urlMigration(formId) {
  return (savedData) => {
    //migration code that you have now, but using formId from above
  };
}
```

Then each form's migration list can look something like `[urlMigration('1990')]`

Reply:
Thanks @jbalboni, that's much simpler, changes made.